qiskit/circuit/parameter.py

```diff
@@ -23,11 +23,11 @@
 class Parameter(ParameterExpression):
     """Parameter Class for variable parameters."""
 
-    def __new__(cls, _, uuid=None):
+    def __new__(cls, name, uuid=None):
+        # pylint:disable=unused-argument
```

Comment:
The ignore should go in the same line, since it should only affect `name` and not the entire function block.
```suggestion
    def __new__(cls, name, uuid=None):  # pylint:disable=unused-argument
```